lib/config/presets/internal/default.ts

```diff
@@ -40,8 +41,9 @@ export const presets: Record<string, Preset> = {
     },
   },
   ignoreModulesAndTests: {
-    description:
-      'Ignore `node_modules`, `bower_components`, `vendor` and various test/tests directories.',
+    description: gt.gettext(
+      'Ignore `node_modules`, `bower_components`, `vendor` and various test/tests directories.'
+    ),
```

Comment:
Why are we applying the addition of that function call selectively? Doesn't it apply to all descriptions? Can't we do it just once, looping over all elements of presets?

(The same question applies for this files' siblings)